src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
+@EntityScan("wolox.training.models")
```

Comment:
I think this annotation shouldn't be necessary. With the `@SpringBootAnnotation` is enough because it will be scan everything inside the wolox.training package (Which is its own package)
If your model classes are outside that package, then it would be necessary to use it